config_generator/src/main.rs

```diff
@@ -47,7 +47,13 @@ struct WranglerConfig {
 fn goto_repository_root() -> Result<(), std::io::Error> {
     let exe_path = std::env::current_exe()?;
     assert!(exe_path.ends_with("target/debug/config-generator"));
-    let repo_root = exe_path.parent().unwrap().parent().unwrap().parent().unwrap();
+    let repo_root = exe_path
+        .parent()
+        .unwrap()
+        .parent()
+        .unwrap()
+        .parent()
+        .unwrap();
```

Comment:
Maybe write this as:
```
let repo_root = exe_path.parent()?.parent()?.parent()?;
```
Since this fn already returns a `Result`.

Reply:
`path.parent()` returns an `Option<T>`, and we can't use question mark here.